SettingsList.py

```diff
@@ -1770,26 +1770,27 @@ def __init__(self, name, gui_text, min, max, default, step=1,
         choices        = {
             'open':       'Always Open',
             'vanilla':    'Vanilla Requirements',
-            'stones':	  'All Spiritual Stones',
-            'medallions': 'All Medallions',
-            'dungeons':   'All Dungeons',
+            'stones':	  'Spiritual Stones',
+            'medallions': 'Medallions',
+            'dungeons':   'Dungeons',
             'tokens':     'Gold Skulltula Tokens'
         },
         gui_tooltip    = '''\
             'Always Open': Rainbow Bridge is always present.
             'Vanilla Requirements': Spirit/Shadow Medallions and Light Arrows.
-            'All Spiritual Stones': All 3 Spiritual Stones.
-            'All Medallions': All 6 Medallions.
-            'All Dungeons': All Medallions and Spiritual Stones.
+            'Spiritual Stones': A configurable amount of 3 Spiritual Stones.
```

Comment:
Nit on the wording here: "amount of 3" feels awkward; would "A configurable amount, up to 3..." make sense?
Or, LACS says "A configurable amount of Spiritual Stones." omitting the number which I think is fine.